src/components/patientdb.js

```diff
@@ -205,34 +205,18 @@ function PatientDB(props) {
           </div>
 
           <div className="select">
-            <select
-              style={{animationDelay: '0.4s'}}
+            <input
+              type="date"
```

Comment:
> input type date has no safari support..I won't recommend using it..may be use one of the react date picker library

Hi @karangarg45

Thanks for the review.

As I mentioned above, I've updated the implementation to have  react-date-picker instead of input type date. This is the outdated commit. Please check the latest.

Thanks :)